siddhi-extension-archetype/siddhi-execution/src/main/resources/archetype-resources/README.md

```diff
@@ -1,4 +1,4 @@
-﻿siddhi-io-
+﻿siddhi-io-${executionType}
```

Comment:
Isn't this for execution extensions? Then the above should be siddhi-execution-{executionType}